mk/cargo.sh

```diff
@@ -108,6 +108,11 @@ case $target in
       export CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="$rustflags_self_contained"
     fi
     ;;
+  loongarch64-unknown-linux-gnuf64)
+    export CC_loongarch64_unknown_linux_gnuf64=clang-$llvm_version
+    export CC_loongarch64_unknown_linux_gnuf64=llvm-ar-$llvm_version
```

Comment:
These need to become `loongarch64-unknown-linux-gnu` after loongson/LoongArch-Documentation#80.